app/image_templates/sip_poll.rb

```diff
@@ -0,0 +1,19 @@
+module ImageTemplates
```

Comment:
Add test for this similar to https://github.com/producthunt/ShareMeow/blob/master/spec/image_templates/question_spec.rb